var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -59,13 +59,17 @@ class Openmpi(AutotoolsPackage):
     """
 
     homepage = "http://www.open-mpi.org"
-    url = "http://www.open-mpi.org/software/ompi/v1.10/downloads/openmpi-1.10.1.tar.bz2"
+    url = "https://www.open-mpi.org/software/ompi/v2.1/downloads/openmpi-2.1.0.tar.bz2"
     list_url = "http://www.open-mpi.org/software/ompi/"
-    list_depth = 3
+    list_depth = 10
```

Comment:
This doesn't look right. `list_depth` is the number of links that you should follow to find versions of openmpi, starting at `list_url`. In this case, 3 and 10 result in the same output from `spack versions`. I would undo this particular change.